solutions/usaco-809.mdx

```diff
@@ -79,4 +79,57 @@ public class Taming {
 				 
 </JavaSection>
 
+<PySection>
+
+```py
+fin = open("taming.in", 'r')
+fout = open('taming.out', 'w')
+
+inputs = [str(i) for i in fin.read().splitlines()]
+days = int(inputs[0])
+entry = [int(i) for i in inputs[1].split(" ")]
+
+time = -1 # Keep tracks of the days left where 0 outbreaks is guaranteed
+outbreaks = 0 # Count the number of outbreaks that must happen
+potential = 0 # Count the number of outbreaks that can potentially happen
+valid = True # Check if the input is valid
+
+# If the first entry is above 0, then we have a contradiction
+if entry[0] > 0:
+	valid = False
+else:
+	entry[0] = 0
+
+"""
+As long as there is no contradiction, we use the rules following:
+1.  If the entry log says a day from now has an outbreak, 
+	then that day must have an outbreak,
+	every other day between them must have no outbreaks.
+2.  Otherwise, if the entry log says -1, 
+	then that day may have a potential outbreak.
+"""
```

Comment:
jeff did you make sure the py code was consistent w/ the c++ & java code?

Reply:
yea, the code's logic is generally the same

Reply:
i'll believe you lol

Reply:
if the c++/java doesn't have it, either
* remove this comment
* or add it to the other solutions as well

Reply:
bump

Reply:
wait nvm you solved it mb mb